.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: minimal
+sudo: required
+dist: xenial
+cache:
+  directories:
+    - /home/travis/.cache/bazel/
+services:
+  - docker
+matrix:
+  include:
+   - os: linux
+     arch: amd64
+     env: RUNSC_PATH=./bazel-bin/runsc/linux_amd64_pure_stripped/runsc
+   - os: linux
+     arch: arm64
+     env: RUNSC_PATH=./bazel-bin/runsc/linux_arm64_pure_stripped/runsc
+script:
+   - uname -a
+   - make DOCKER_RUN_OPTIONS="" BAZEL_OPTIONS="build runsc:runsc" bazel && $RUNSC_PATH --alsologtostderr --network none --debug --TESTONLY-unsafe-nonroot=true --rootless do ls
```

Comment:
Optional: can you add a basic smoke test to scripts/make_test.sh and invoke that? That way if we update that smoketest it will apply to all relevant workflows.

Reply:
(And then you can also drop the RUNSC_PATH bits above!)

Reply:
Let's do this step by step. I want to start with compiling runsc and running a simple test. Then I will work adding compilation of tests and then we will be able to switch to make_test.sh.